src/engine/GameState.java

```diff
@@ -155,6 +155,9 @@ public final Coin setCoin(final Coin coin) {
 	public final double getLivesRemaining() {
 		return livesRemaining;
 	}
+	public final void plusLives() {
+		this.livesRemaining += 1;
+	}
```

Comment:
Codacy Static Code Analysis — 1 new issue (0 max.) of at least minor severity. 

'METHOD_DEF' should be separated from previous line.